git/cmd.py

```diff
@@ -1210,6 +1210,14 @@ def execute(
         if self.GIT_PYTHON_TRACE and (self.GIT_PYTHON_TRACE != "full" or as_process):
             _logger.info(" ".join(redacted_command))
 
+        if strip_newline_in_stdout and command[:2] == ["git", "show"]:
```

Comment:
`"git"` shouldn't be hard-coded, and I am pretty sure there is a variable for this somewhere.